blockly-core/core/utils/utils.js

```diff
@@ -754,6 +754,8 @@ Blockly.svgRectToRect = function (svgRect) {
  * @typedef {Object} DialogOptions
  * @property {string} headerText
  * @property {string} bodyText
+ * @property {boolean} prompt
+ * @property {string} promptPrefill
```

Comment:
Hmmm. A cursory glance indicates that `Blockly.ShowSimpleDialog` is used exclusively by `Feedback.showSimpleDialog` ... if that's actually the case, I'd say it makes sense to remove this JSDoc typedef (or move it to feedback) and replace it with an `@see`